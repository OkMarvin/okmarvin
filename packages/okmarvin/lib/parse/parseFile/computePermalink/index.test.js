const fn = require('./index')
test('returns permalink', () => {
  expect(
    fn(
      '/:month/:title',
      { categories: [], title: 'okmarvin' },
      new Date('2010-10-10'),
      'content/page/hello'
    )
  ).toBe('/10/okmarvin/')
})
