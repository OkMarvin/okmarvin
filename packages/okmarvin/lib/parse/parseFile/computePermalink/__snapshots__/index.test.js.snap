// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`returns permalink 1`] = `"/10/okmarvin/"`;
